src/components/HighlightedText/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import HitText from './hitText';
+import { DefaultHitContainer, DefaultTextContainer } from './styled/index';
+
+/**
+ * HighlightedText is a component that highlights a part of a text.
+ */
+
+function HighlightedText(props) {
+    const { style, className, parts, hitComponent, textComponent } = props;
+    const finalHitContainer = hitComponent || DefaultHitContainer;
+    const finalTextContainer = textComponent || DefaultTextContainer;
+
+    return (
+        <p className={className} style={style}>
```

Comment:
We need to add a new prop `isInline` in order to change the `p` element by a `span` element, also we should add a useful description for this prop

Reply:
add a new example using the `isInline` prop